### DISCUSSION TITLE
模拟冬小麦时候遇到的问题。

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#200] [Q&A] [answered]
您好，我现在用您的作物模型想跑冬小麦的生长情况，比如我想模拟冬小麦生长开始时间为2020年9月份，第243天，结束时间为第二年2021年的6月份，第212天，但是在用extract_gldas.py代码时，只能做2020年的，有什么建议吗？
<img width="362" alt="extract_gldas py里面修改日期的代码" src="https://github.com/user-attachments/assets/f1a886c8-7063-4b76-9b98-59eed9d4ad64">

**Comment by cenlinhe** [comment, marked as answer]
@CharlesZheZhang Could you please take a look at this issue?
Note that currently Noah-MP-Crop physics does not include some key processes (or growth stages) for winter wheat. We have plans to improve that in the future, but at this stage you will have to include those processes by yourself if you want.

**Reply by 12wlj** [reply]
谢谢您

**Comment by ZhaoY-github** [comment]
You might want to check out this paper, it's already accepted on GMD
https://essopenarchive.org/users/667573/articles/667753-integrating-a-double-cropping-model-with-groundwater-fed-irrigation-in-the-north-china-plain

**Reply by 12wlj** [reply]
谢谢您，我在看这篇论文。
